app/commands.py

```diff
@@ -84,3 +85,26 @@ def post_tweet_command():
         exit(0)
     else:
         exit(1)
+
+
+@app.cli.command('cleanup-db')
+def cleanup_db():
+    """Deletes older partition than 14 days. Executed weekly via cron"""
+
+    eng = db.engine
+
+    eng.execute(
+        "DELETE FROM ockovani_registrace WHERE import_id<(SELECT coalesce(min(id), 5) FROM importy WHERE start>now()-'14 days'::interval)")
```

Comment:
5 is there just for the case when no row is returned

Reply:
it should be possible to delete just the import and use cascade

Reply:
null shouldn't cause any issue since `import_id < null` is false, or not?